content/applications/marketing/events/lead_generation_rules.rst

```diff
@@ -0,0 +1,206 @@
+Generation`), then click :guilabel:`New`.
+
+Doing so reveals a blank :guilabel:`Lead Generation Rule` form.
+
+.. image:: lead_generation_rules/lead-generation-rule-form.png
+   :align: center
+   :alt: A standard Lead Generation Rule form in the Odoo Events application.
+
+First, start by entering a :guilabel:`Rule Name`.
+
+Then, designate a lead creation option for the lead generation rule in the :guilabel:`Create` field.
+The options are: :guilabel:`Per Attendee` or :guilabel:`Per Order`.
+
+If :guilabel:`Per Attendee` is selected, a lead is created for each attendee (B2C). If
+:guilabel:`Per Order` is selected, a single lead is created per ticket batch/sales order (B2B).
+
+Next, designate *when* the lead should be created in the :guilabel:`When` field. The options are:
+
+- :guilabel:`Attendees are created`: leads that follow the configurations of the rule are created
+  when an attendee record for an event is created.
+- :guilabel:`Attendees are registered`: leads that follow the configurations of the rule are created
+  when the attendee officially registers, either manually or automatically.
+- :guilabel:`Attendees attended`: leads that follow the configurations of the rule are created when
+  the attendee's attendance is officially confirmed, and the registration is set to *Done*.
+
+For Any Of These Events section
+-------------------------------
+
+In the :guilabel:`For Any Of These Events` section of the :guilabel:`Lead Generation Rule` form that
+follows, there are a few options to configure.
+
+First, users have the option to filter the attendees to include those who have attended specific
+event categories in the :guilabel:`Event Categories` drop-down field. Multiple :guilabel:`Event
+Categories` can be selected.
+
+If no :guilabel:`Event Categories` are selected, no event category restriction is applied to the
+lead generation rule.
+
+Next, in the :guilabel:`Company` field, users can choose to restrict the triggering of this lead
+generation rule to events belonging to a specific company (if working in a multi-company
+environment). If left blank, no company restriction is applied.
+
+Lastly, in the :guilabel:`Event` field, choose to filer the attendees to include those of a specific
+event, via the drop-down menu. If left blank, no event restriction is applied.
+
+If The Attendees Meet These Conditions section
+----------------------------------------------
+
+In the :guilabel:`If The Attendees Meet These Conditions` section of the :guilabel:`Lead Generation
+Rule` form, users can configure specific criteria for their rule.
+
+By default, this field is set to :guilabel:`Match all records` of attendees in the database.
+
+To add specific criteria, click the :icon:`fa-plus` :guilabel:`Add condition` button. Or, click the
+:icon:`fa-caret-right` :guilabel:`(right arrow)` icon to the left of :guilabel:`Match all records`.
+
+Clicking the :icon:`fa-plus` :guilabel:`Add condition` button reveals a pre-configured condition
```

Comment:
```suggestion
Clicking the :icon:`fa-plus` :guilabel:`Add condition` button reveals a preconfigured condition
```
I believe that the consensus was preconfigured _without_ a hyphen (the linter has not been updated yet)